doc/cascadia/profiles.schema.json

```diff
@@ -298,6 +307,22 @@
       ],
       "required": [ "commandline" ]
     },
+    "TabSwitcherAction": {
+      "description": "Arguments corresponding to a Tab Switcher Action",
+      "allOf": [
+        { "$ref": "#/definitions/ShortcutAction" },
+        {
+          "properties": {
```

Comment:
Maybe add a "direction" for going forwards/backwards as well? Otherwise if someone binds this to <kbd>ctrl+tab</kbd>, they'd expect <kbd>ctrl+shift+tab</kbd> to open the tab switcher and have the currently focused tab - 1 to be focused.

Reply:
donezo